jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,71 @@
+const books = require('./books.json')
```

Comment:
You're missing a semicolon here.

Is your HoundCI working? I surprised it's not flagging this.